iroh/examples/transfer.rs

```diff
@@ -265,9 +265,9 @@ impl EndpointArgs {
 
         #[cfg(feature = "qlog")]
         {
-            let mut transport_config = iroh::endpoint::QuicTransportConfig::default();
-            transport_config.qlog_from_env("transfer");
-            builder = builder.transport_config(transport_config)
+            let mut cfg_builder = iroh::endpoint::QuicTransportConfigBuilder::default();
+            cfg_builder.qlog_from_env("transfer");
```

Comment:
I think you can now chain this easily:
```rust
let cfg_builder = iroh::endpoint::QuicTransportconfigBuilder::new()
    .qlog_from_env("transer");
```
(I'm assuming `new` exists there) This is totally nitpicking, but for examples trying to write the most idiomatic thing might be worth it.